packages/roosterjs-content-model-core/lib/coreApi/setContentModel/setContentModel.ts

```diff
@@ -15,6 +18,18 @@ import type { SetContentModel } from 'roosterjs-content-model-types';
  */
 export const setContentModel: SetContentModel = (core, model, option, onNodeCreated) => {
     const editorContext = core.api.createEditorContext(core, true /*saveIndex*/);
+    const currentWindow = core.logicalRoot.ownerDocument.defaultView;
+    if (currentWindow && isWindowWithHighlight(currentWindow)) {
```

Comment:
I feel we should check "!" for isWindowWithHighlight(), we only need this CSS based solution when browser does not support highlight, right?